data/mods/Aftershock/maps/crashing_ship.json

```
@@ -0,0 +1,184 @@
+[
+  {
+    "type": "palette",
+    "id": "aftershock_crashing_ship",
+    "terrain": { "0": "t_floor_warped", "1": "t_wall_warped", "a": "t_grass_alien", "G": "t_grass" }
+  },
+  {
+    "//": "Aftershock crashing ship, divided into six sections. This is intended to be the port side of a larger ship, with the corridors leading to the rest of the ship sealed off due to hull breaches.",
+    "//": "Barracks (Top Left): Top Left: locker room, meant to store work clothes and small tools/trinkets. Top Right: armory, stores weapons and armor. Mid Left: Bathroom/showers. Mid Right: Sleeping quarters. Lockers in corner will have emergency supplies. Bottom: Cafeteria.",
```

```suggestion
    "//1": "Barracks (Top Left): Top Left: locker room, meant to store work clothes and small tools/trinkets. Top Right: armory, stores weapons and armor. Mid Left: Bathroom/showers. Mid Right: Sleeping quarters. Lockers in corner will have emergency supplies. Bottom: Cafeteria.",
```
You can number these so it doesn't error.